docs/weightonlyquant.md

```diff
@@ -135,4 +140,82 @@ loaded_model = AutoModelForCausalLM.from_pretrained(saved_dir)
 |       LLM Runtime (use_llm_runtime=True)      |  &#10004;  |  &#10004;  |
 |       PyTorch (use_llm_runtime=False)      |  stay tuned  | &#10004; |
 
-> Note: Only supports CPU device for now. For LLM runtime model loading usage, please refer to [graph readme](../intel_extension_for_transformers/llm/runtime/graph/README.md#2-run-llm-with-transformer-based-api)
+> Note: For LLM runtime model loading usage, please refer to [graph readme](../intel_extension_for_transformers/llm/runtime/graph/README.md#2-run-llm-with-transformer-based-api)
+
+## Examples For GPU
+Intel-extension-for-transformers implement weight-only quantization for intel GPU(PVC and ARC) with Intel-extension-for-pytorch. Currently, the Linear op kernel of Weight-only quantization is implemented in the Intel-extension-for-pytorch branch: "dev/QLLM".  

```

Comment:
add this sentence:
We support experimental woq inference on intel GPU(PVC and ARC) with replacing Linear op in PyTorch. 

and add the link of ipex